site/src/test/resources/logback-test.xml

```diff
@@ -1,14 +0,0 @@
-<configuration>
```

Comment:
I don't really understand what this had to do with the site generation at all, esp. since it's in the `test` directory. So out it goes.

Reply:
I suspect this cleaned up some console noise while the site is being built, but I can't explain the test directory.

Reply:
Well, if it ever did, it stopped working.  The SLF4J noise is already on the branch.